ktor-network/ktor-network-quic/jvmAndNix/src/io/ktor/network/quic/connections/QUICConnection_v1.kt

```diff
@@ -726,6 +762,89 @@ internal class QUICConnection_v1(
         private val logger = logger()
     }
 
+    private inner class StreamManager {
+        private val receiveStates = hashMapOf<Long, ReceiveStreamState>()
+
+        private val streams = hashMapOf<Long, QUICStreamImpl>()
+
+        private val sendChannel: Channel<StreamFrame> = Channel(Channel.UNLIMITED)
+
+        init {
+            // todo close
+            CoroutineScope(Dispatchers.Default).launch {
+                while (isActive) {
```

Comment:
Could you tell me why it's needed?

Reply:
From the `isActive` doc:

Check this property in long-running computation loops to support cancellation:
```kotlin
while (isActive) {
     // do some computation
}
```

Reply:
I know that I don't close it though :) But is does no harm here, and may prevent future me from forgetting to add it when I support cancelation.